releasenotes/notes/docker-io-per-device-92047e9e294be1a6.yaml

```diff
@@ -0,0 +1,3 @@
+---
+enhancements:
+  - Docker disk IO metrics are now tagged by ``device``
```

Comment:
are the double backquote expected?

Reply:
Yup, reno uses [reStructuredText](https://github.com/DataDog/datadog-agent/blob/master/docs/dev/contributing.md#reno)